migrations/20161213112854_add-claim-column-total-amount.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.table('Claim', function (table) {
+    table.decimal('TotalAmount')
```

Comment:
- rename PaymentAmount to BankPaymentAmount
- add another column ManuallyProcessedAmount

So three columns (to be clear in MI), TotalAmount, BankPaymentAmount, ManuallyProcessedAmount